<launch>
<arg name="task_scheduler"/>

<group unless="$(eval task_scheduler == 'None')">
    <node name="task_scheduler" pkg="int_preservation" type="$(arg task_scheduler).py">
    </node>
</group>

</launch>